src/components/Courses/Courses.jsx

```diff
@@ -64,17 +61,21 @@ export default function Courses() {
 			{courses.length === 0 ? (
 				<div>There are no courses</div>
 			) : (
-				filteredCourses.map((course) => (
-					<CourseCard
-						key={course.id}
-						id={course.id}
-						title={course.title}
-						description={course.description}
-						authors={course.authors}
-						duration={course.duration}
-						creationDate={course.creationDate}
-					/>
-				))
+				filteredCourses.map((course) =>
+					!course.id ? (
+						'loading...'
+					) : (
```

Comment:
Once you add getCourse info on add/delete/update you won't need this

Reply:
Understood.